Bigtable/src/Instance.php

```diff
@@ -217,8 +182,15 @@ public function state(array $options = [])
      *
      * Example:
      * ```
-     * $instanceType = Instance_Type::PRODUCTION
-     * $operation = $instance->create('instanceId', $instanceType, ['foo'=>'bar'], $bigtable->clusterMetadata());
+     * use Google\Cloud\Bigtable\BigtableClient;
```

Comment:
You don't keep consistency here.
Why import only BigtableClient but not Instance below?
I'd add import for Instance too or not do import at all.